src/app/services/exchanges/binance_exchange_service.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+from datetime import datetime
+import sqlalchemy_get_or_create
+
+from app.models import Candlestick, CurrencyPair, Exchange, Currency
+
+from app.services.exchanges.exchange_service_base import ExchangeServiceBase
+
+
+class BinanceExchangeService(ExchangeServiceBase):
```

Comment:
I believe that the methods in this class could be ported to `ExchangeServiceBase` one instead, leaving just the `EXCHANGE_CODE` (and perhaps the `EXCHANGE_NAME`) on this one. Of course, we could override them if needed, but I don't see why we'd do it here.

Reply:
Done